src/applications/personalization/profile/components/connected-apps/ConnectedApps.jsx

```diff
@@ -132,11 +133,10 @@ export class ConnectedApps extends Component {
             <AlertBox
               key={`${app.attributes?.title}`}
               className="vads-u-margin-bottom--2"
-              headline={`We couldn’t disconnect ${app.attributes?.title}`}
-              status="error"
-              content={`We’re sorry. Something went wrong on our end and we couldn’t disconnect ${
+              status={AlertBoxComponent.ALERT_TYPE.ERROR}
+              content={`We’re sorry. We can't disconnect this app from your VA.gov profile right now: ${
```

Comment:
you've got some straight single quotes here that should be single curly `’`

Reply:
Not sure I fully understand. These are single curly. Done the same as what I'm replacing, I believe.

Reply:
Should be `can’t` instead of `can't`. It's a subtle difference that's more obvious when rendered with the website's font.

Reply:
if you zoom in you can see the difference between curly and straight 😆
<img width="474" alt="Screen Shot 2021-12-01 at 11 24 43 AM" src="https://user-images.githubusercontent.com/20728956/144300141-7df82e9a-9494-4bb0-863e-82185d8a2df1.png">

Reply:
oh Sorry about that. I did screw that up. Will fix

Reply:
I need glasses clearly. lol